lib/utils.js

```diff
@@ -1,6 +1,14 @@
 var osHomeDir = require('os-homedir')();
 var path = require('path');
 
+var osascript = function(script) {
```

Comment:
Ok. Moved everything regarding terminal interaction from `utils.js`  to `common/terminal.js`.